css/style.css

```diff
@@ -0,0 +1,31 @@
+*{margin: 0;padding: 0;box-sizing: border-box;}
+body{background-color: gray;}
+header,footer{height: 170px;
+    width: 97vw;
+    background: rgb(216, 214, 214);
+   
+    color: rgb(17, 16, 16);
+    text-align: center;
+    margin-right: 2rem;
+margin-left: 1rem;
+margin-top: 1rem;}
+header>h1{padding-top: 3.5rem;}
+footer>h3{padding-top: 3.5rem;}
+.aboutme img{height: 100px;}
+
+.main{display: grid;
+grid-template-columns: 1fr 300px;
+grid-template-rows: 200px 200px 200px 150px 150px 100px 200px;}
+header{grid-row: 1/2;grid-column: 1/3;}
+.blog1{display: flex;flex-direction: column;padding: 1rem;background-color: white;grid-row: 2/4;grid-column: 1/2;margin-left: 1rem;}
```

Comment:
blog1 and blog2 looks identical, no need to write twice, that keeps code concise. No need for flex in here.